.ci_support/build_all.py

```diff
@@ -60,6 +60,7 @@ def build_all(recipes_dir, arch):
 
     deployment_version = (0, 0)
     sdk_version = (0, 0)
+    channel_urls = ['local']
```

Comment:
Suggestion: this could be a set() with conda-forge already added:
`channel_urls = {'local', 'conda-forge'}`

Reply:
No, sets are unordered.

Reply:
True, thanks for the correction, what about something like this:
```
    channel_sources = []
    for folder in folders:
            [...]
            if 'channel_sources' in text:
                config = load(text, Loader=BaseLoader)
                new_channel_sources = config['channel_sources'][0].split(',')
                if len(channel_sources) > 0 and set(new_channel_sources).difference(channel_sources) > 0:
                    raise ValueError("All recipes should have the same values for channel_sources.")
                channel_sources.extend(channel_source)

    channel_urls = ['local'] + channel_sources
```